docs/license.rst

```diff
@@ -0,0 +1,14 @@
+License
+=======
+
```

Comment:
Should include the line `Copyright 2015 Lifealike Ltd ` here. (see the appendix to the license)